draft-west-cookie-incrementalism.md

```diff
@@ -224,6 +233,59 @@ attribute (Section 4.1.2.5 of {{RFC6265bis}}) by altering the storage model defi
 This is conceptually similar to the requirements put into place for the `__Secure-` prefix (Section
 4.1.3.1 of {{RFC6265bis}}).
 
+## Schemeful Same-Site {#schemeful-samesite}
+
+By using the scheme, as well as the registrable domain, SameSite can help to
+protect https origins against a network attacker that is impersonating an http
+origin with the same registrable domain. Further increasing its CSRF
+protections. To do so we need to modify a number of things:
+
+First change the definition of "site for cookies" from a registrable domain to
+an origin. In the places where a we return an empty string for a non-existent
+"site for cookies" we should instead return an origin set to a freshly
+generated globally unique identifier.
+
+Then replace the same-site calculation algorithm with the following
+~~~
```

Comment:
Hrm, does this look right when you build this? `~~~` makes it just draw ASCII art and not interpret the contents as markdown at all.

Reply:
Oh right, the whole building thing... I'll check once I get a commit uploaded for the other issues.

Reply:
It seems fine once built
The .txt version just has the same text, the .html version puts its in a code block.

In the .txt version the "A request is 'same-site'..." portion looks like it's not part of the monkey patch which may be confusion however.

Reply:
See my above comment RE merging this PR.